packages/yourloops/test/integration/assert/patient-list.assert.ts

```diff
@@ -755,10 +757,58 @@ const checkDefaultColumnsDisplay = () => {
   expect(screen.getByRole('columnheader', { name: 'Monitoring alerts' })).toBeVisible()
   expect(screen.queryByRole('columnheader', { name: 'System' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Messages' })).toBeVisible()
-  expect(screen.getByRole('columnheader', { name: 'Time In Range' })).toBeVisible()
-  expect(screen.queryByRole('columnheader', { name: 'GMI (estimated HbA1c of last 14 days)' })).not.toBeInTheDocument()
+  expect(screen.getByRole('columnheader', { name: 'TIR' })).toBeVisible()
+  expect(screen.queryByRole('columnheader', { name: 'GMI' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Below range' })).toBeVisible()
-  expect(screen.queryByRole('columnheader', { name: 'Coefficient of Variation' })).not.toBeInTheDocument()
+  expect(screen.queryByRole('columnheader', { name: 'CV' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Last data update' })).toBeVisible()
   expect(screen.getByRole('columnheader', { name: 'Actions' })).toBeVisible()
 }
+
+const checkTooltipsColumnHeader = async (dataGridRows) => {
+  const tooltipTextMonitoringAlerts = 'Hover over the icons to learn more'
+  const tooltipTextMessages = 'Messages'
+  const tooltipTextBelowRange = 'Below Range'
+  const tooltipTextManagementGlucose = 'GMI (estimated HbA1c)'
+  const tooltipTextTimeInRange = 'Time in range'
+  const tooltipTextVariant = 'Coefficient of Variation'
+  const tooltipTextLastDataUpdate = 'Last data Update'
+
+  const monitoringAlertsColumnHeader = within(dataGridRows).getByText('Monitoring alerts')
+  const messagesColumnHeader = within(dataGridRows).getByText('Messages')
+  const TimeInRangeColumnHeader = within(dataGridRows).getByText('TIR')
+  const managementGlucoseColumnHeader = within(dataGridRows).getByText('GMI')
+  const belowRangeColumnHeader = within(dataGridRows).getByText('Below range')
+  const variantColumnHeader = within(dataGridRows).getByText('CV')
+  const LastDataUpdateColumnHeader = within(dataGridRows).getByText('Last data update')
+
+  await userEvent.hover(monitoringAlertsColumnHeader)
+  expect(await screen.findByText(tooltipTextMonitoringAlerts)).toBeVisible()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏I don't think the variables for the tooltip texts are very useful, I feel that when using the `getByText()/findByText()` methods, we usually provide the raw string rather than putting the string in a variable. It will allow the test to have less lines and (in this case) be more readable